www/manual_lib/enketo/css/plain-print.css

```diff
@@ -0,0 +1,332 @@
+/** CORE **/
```

Comment:
We are not using the enketo embedded forms at this point, so none of the enketo `www/manual_lib` stuff is required.